docs/triggerbindings.md

```diff
@@ -10,11 +11,37 @@ kind: TriggerBinding
 metadata:
   name: pipeline-binding
 spec:
-  params:
+  inputParams:
+    - name: message
+      description: The message to print
+      default: This is the default message
```

Comment:
Do you think it makes sense to have default within the `TriggerTemplate`? Since these _should_ be specified in the `EventListener`, maybe better illustrates the behavior that these should be passed into the `TriggerBinding`.

Reply:
> Do you think it makes sense to have default values for inputParams within the TriggerTemplate?

(I'm assuming this meant `TriggerBinding`) This is a good point; I didn't really think about whether the user would need a `default` value, I used them for the sake of consistency. I worry that omitting `default` might be confusing if these parameter specs behave differently than the other parameter specs in Tekton.

Also, since the `default` value is optional, I think it is alright to include it.